docs/nightly/en/getting-started/quick-start.md

```diff
@@ -0,0 +1,382 @@
+# Quick Start
+
+Before proceeding, please ensure you have [installed GreptimeDB](./installation/overview.md).
+
+This guide will walk you through creating a metric table and a log table, highlighting the core features of GreptimeDB.
+
+## Connect to GreptimeDB
+
+GreptimeDB supports [multiple protocols](/user-guide/clients/overview.md) for interacting with the database.
+In this quick start document, we use SQL for simplicity.
+
+If your GreptimeDB instance is running on `127.0.0.1` with the MySQL client default port `4002` or the PostgreSQL client default port `4003`,
+you can connect to it using the following commands:
+
+```shell
+mysql -h 127.0.0.1 -P 4002
+```
+
+Or
+
+```shell
+psql -h 127.0.0.1 -p 4003 -d public
+```
+
+## Create tables
+
+Suppose you have a table named `grpc_latencies` that stores the gRPC latencies of your application.
+The table schema is as follows:
+
+```sql
+CREATE TABLE grpc_latencies (
+  ts TIMESTAMP TIME INDEX,
+  host STRING,
+  method_name STRING,
+  latency DOUBLE,
+  PRIMARY KEY (host, method_name)
+)
+engine=mito with('append_mode'='true');
+```
+
+- `ts`: The timestamp when the metric was collected. It is the time index column.
+- `host`: The hostname of the application server. It is a tag column.
+- `method_name`: The name of the RPC request method. It is a tag column.
+- `latency`: The latency of the RPC request.
+
+Additionally, there is a table `app_logs` for storing application logs:
+
+```sql
+CREATE TABLE app_logs (
+  ts TIMESTAMP TIME INDEX,
+  host STRING,
+  api_path STRING FULLTEXT,
```

Comment:
Is it still necessary to use `FULLTEXT`? Maybe making `log` `FULLTEXT` is enough.

Reply:
I believe that's necessary. You might want to query APIs inaccurately.